src/openforms/config/models/map.py

```diff
@@ -0,0 +1,35 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+
+class MapTileLayer(models.Model):
+    identifier = models.SlugField(
+        _("identifier"),
+        unique=True,
+        max_length=50,
+        help_text=_("A unique identifier for the tile layer"),
+    )
+    url = models.URLField(
+        _("tile layer url"),
+        max_length=255,
+        help_text=_(
```

Comment:
I think you need a disclaimer that the tiler layers must use EPSG:28992 (Rijksdriehoek coordinate system), using other systems will probably lead to weird results.